CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -171,6 +173,7 @@ public function upgrade_4_7_2($rev) {
    * @param string $rev
    */
   public function upgrade_4_7_3($rev) {
+    $this->addTask(ts('Upgrade DB to %1: SQL', array(1 => $rev)), 'runSql', $rev);
```

Comment:
In case you're wondering if the above 3 additions are non-functional, they are, since I've removed the contents of those sql files. I just did this for the sake of setting a good example.